ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/intended/configs/SITE3-LER1.cfg

```diff
@@ -69,8 +69,6 @@ mpls ldp
    no shutdown
    transport-address interface Loopback0
 !
-patch panel
-!
```

Comment:
Change is a fix of unneeded patch panel key created even if no patches are defined.